Measurement_2015.py

```diff
@@ -113,7 +113,7 @@ def DoDeviceSweep(GraphProc,rpg,DataFile,SweepInst,ReadInst,
 	
 	NowTime = datetime.now()
 	Remaining = Timeout*60.0 - float((NowTime-SetTime).seconds)
-	while (TSocket[1] != 1) and (Remaining > 0):
+	while (TSocket[4] != 1) and (Remaining > 0):
```

Comment:
Is this also a difference between 16T and DF? Check v0 and v2 to find out later.